src/data/stig_multipliers.ts

```diff
@@ -1550,6 +1628,60 @@ const DPS_STIG_MULTIPLIERS = Object.freeze({
       }),
     ],
   },
+  "Shattered Swords": {
+    T: [
+      new Multiplier({
+        active: true,
+        name: "Raksha T",
```

Comment:
Same comment as above, let's just use Shattered Swords T/M/B for the description since the button people click on just says T/M/B

Reply:
Renamed `Raksha`, `Sushang`, and `Empyrea Phoenix` to `Shattered Swords`